app/lib/world/world.coffee

```diff
@@ -260,6 +260,11 @@ module.exports = class World
         system.start @thangs
       catch err
         console.error "Error starting system!", system, err
+    # Silently clamp health after systems run.
+    hero = _.find(@thangs, (thang) -> thang.id == 'Hero Placeholder')
```

Comment:
Not sure if this is the best way to locate the hero.

Reply:
This works. You can also shorten a bit:
```
    hero = _.find @thangs, id: 'Hero Placeholder'
```